src/components/scenes/Loans/ManageCollateralScene.tsx

```diff
@@ -41,51 +40,52 @@ import { NetworkFeeTile } from '../../tiles/NetworkFeeTile'
 import { TotalDebtCollateralTile } from '../../tiles/TotalDebtCollateralTile'
 import { FormScene } from '../FormScene'
 
-type ActionOpType = 'loan-borrow' | 'loan-deposit' | 'loan-repay' | 'loan-withdraw'
+export type LoanManageActionOpType = 'loan-deposit' | 'loan-withdraw' | 'loan-borrow' | 'loan-repay'
 
 // User input display strings
 const ACTIONOP_TYPE_STRING_MAP = {
-  'loan-borrow': {
-    headerText: s.strings.loan_borrow_more,
-    amountCard: s.strings.loan_fragment_loan,
-    srcDestCard: s.strings.loan_fund_destination
-  },
   'loan-deposit': {
     headerText: s.strings.loan_add_collateral,
     amountCard: s.strings.loan_fragment_deposit,
     srcDestCard: s.strings.loan_fund_source
   },
-  'loan-repay': {
-    headerText: s.strings.loan_make_payment,
-    amountCard: s.strings.loan_fragment_repay,
-    srcDestCard: s.strings.loan_fund_source
-  },
   'loan-withdraw': {
     headerText: s.strings.loan_withdraw_collateral,
     amountCard: s.strings.loan_fragment_withdraw,
     srcDestCard: s.strings.loan_fund_destination
+  },
+  'loan-borrow': {
+    headerText: s.strings.loan_borrow_more,
+    amountCard: s.strings.loan_fragment_loan,
+    srcDestCard: s.strings.loan_fund_destination
+  },
+  'loan-repay': {
+    headerText: s.strings.loan_make_payment,
+    amountCard: s.strings.loan_fragment_repay,
+    srcDestCard: s.strings.loan_fund_source
```

Comment:
This re-ordering change should be squashed into the previous commit.

Reply:
Force pushed these changes